avocado/utils/software_manager.py

```diff
@@ -484,9 +484,7 @@ def __init__(self, cmd='yum'):
         Initializes the base command and the yum package repository.
         """
         super(YumBackend, self).__init__()
-        executable = utils_path.find_command(cmd)
```

Comment:
`executable` is still used on line 491.
I continue reviewing since it is likely you fixed it later another commit :)

Reply:
Indeed, `executable` is removed later. It is up to you @clebergnu , change the first commit or simply merge the series (all remaining commits look good to me).

Reply:
True, I missed that.  I'm replacing the `executable` variable with `self.base_command` and sending a v2.